frontend/views/accessions/index.html.erb

```diff
@@ -2,10 +2,12 @@
 
 <%
   # Start of modifications
+  # Customize accession browse columns
   # See https://github.com/archivesspace/archivesspace/blob/master/frontend/app/helpers/search_helper.rb
   model = "accession"
+  add_multiselect_column if can_delete_search_results?(model) && !(request.path =~ /\/(advanced_)*search/)
```

Comment:
I realized this was missing and the user could not select rows to delete multiple records at once (though they could delete them individually from the "View" pages). Another regression 😞